### ISSUE TITLE
v4-alpha docs should note removal of non-responsive float utility classes

### ISSUE THREAD

**Issue by Libbux** [twbs/bootstrap#18636]
The v4 docs do not currently include any warning of the fact that the `pull-left` and `pull-right` utility classes no longer exist, and have been replaced by the responsive float utilities. This is a relatively major breaking change, so the docs should warn users somewhere.

Here's an [example] (http://codepen.io/anon/pen/dGXrPg) of the change for anyone who lands here from Google.

**Comment by twbs-lmvtfy** [comment]
Hi @Libbux!

You appear to have posted a live example (http://codepen.io/anon/pen/dGXrPg.html), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 11, column 5: [W009](https://github.com/twbs/bootlint/wiki/W009): Using empty spacer columns isn't necessary with Bootstrap's grid. So instead of having an empty grid column with `class="col-md-5"` , just add `class="col-md-offset-5"` to the next grid column.
* line 20, column 5: [W009](https://github.com/twbs/bootlint/wiki/W009): Using empty spacer columns isn't necessary with Bootstrap's grid. So instead of having an empty grid column with `class="col-md-5"` , just add `class="col-md-offset-5"` to the next grid column.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by Libbux** [comment]
Note about the above automated response: the empty columns are necessary to illustrate this example.

**Comment by cvrebert** [comment]
The removal/replacement is already covered in the migration guide: http://v4-alpha.getbootstrap.com/migration/#utilities
